src/main/java/presenter/FinancePresenter.java

```diff
@@ -0,0 +1,5 @@
+package presenter;
+
+public class FinancePresenter {
```

Comment:
should we do one presenter for each panel?

Reply:
same for this - making separate branch for presenters